companies/arcteryx.md

```diff
@@ -0,0 +1,25 @@
+---
+fortune_rank: number
+name: Arc'teryx
+tags: ["fortune500", "sports"]
+image: arcteryx.jpeg
+blm_statements:
+  - url: https://www.instagram.com/p/CA5v5wYJsb0/
+    date_posted: 2020-06-01
```

Comment:
```suggestion
    date_posted: 2020-06-01
    type: Instagram
```

I recently added a `type` field for the statements. I haven't been able to update all the existing data & add it to the instructions yet.